packages/redux-store/src/slices/connections/connectionsSlice.ts

```diff
@@ -39,6 +37,9 @@ const connectionsSlice = createSlice({
   name: 'connections',
   initialState,
   reducers: {
+    setConnections: (state, action: PayloadAction<ConnectionRecord[]>) => {
```

Comment:
Maybe we should call this `setConnectionRecords`. This way it's more consistent when we want to do `setMediations` (as it could be `setMediationRecords`)

Reply:
Fixed in f295b5c